openapi.yaml

```diff
@@ -28,6 +30,7 @@ paths:
               $ref: '#/components/schemas/user'
       tags:
         - user-service
+        - admin
```

Comment:
Is this to signify that only admins (according to their user role) may call this endpoint?

Reply:
This endpoint might not be called from the outside at all but I will check again with the rest of the team.

Reply:
The idea is that clients calling endpoints with these tags will have to be authorised with an appropriate role, yes. Users, under normal circumstances should not be able to call this.